GameServerLib/Chatbox/Commands/PacketCommand.cs

```diff
@@ -38,15 +39,15 @@ public override void Execute(Peer peer, bool hasReceivedArguments, string argume
                 {
                     if (s[i].Equals("netid"))
                     {
-                        packet.Write(_playerManager.GetPeerInfo(peer).Champion.NetId);
+                        packet.Write(_playerManager.GetPeerInfo(userId).Champion.NetId);
```

Comment:
Maybe it would be better if commands were part of PacketDefinitions or if we removed Packet command?

Reply:
We just need sort of command request API type (sort of `RequestCommand(string command)`), I think that commands should be in server lib.